lte/gateway/python/scripts/subscriber_cli.py

```diff
@@ -41,8 +42,21 @@ def add_subscriber(client, args):
     if args.lte_auth_opc is not None:
         lte.auth_opc = bytes.fromhex(args.lte_auth_opc)
 
+    # As of now we support only 2 APNs
+    if args.apn_name1 is not None:
+        apn_config = sub_data.apn_config.add()
+        apn_config.service_selection = args.apn_name1
+    if args.qci1 is not None:
+        apn_config.qos_profile.class_id = args.qci1
+
+    if args.apn_name2 is not None:
+        apn_config = sub_data.apn_config.add()
+        apn_config.service_selection = args.apn_name2
+    if args.qci2 is not None:
+        apn_config.qos_profile.class_id = args.qci2
+
```

Comment:
This does not look elegant. Why don't we instead introduce add-apn, delete-apn, modify-apn type commands for a given subscriber?